target/linux/generic/config-5.10

```diff
@@ -2123,7 +2123,6 @@ CONFIG_GPIOLIB_FASTPATH_LIMIT=512
 # CONFIG_GPIO_MAX732X is not set
 # CONFIG_GPIO_MB86S7X is not set
 # CONFIG_GPIO_MC33880 is not set
-# CONFIG_GPIO_MCP23S08 is not set
```

Comment:
The change for kernel 5.15 is missing

Reply:
@hauke Thank you for pointing that out, I had overlooked it.

Fixed!